Jenkinsfile

```diff
@@ -4,23 +4,40 @@ pipeline {
     timeout(time: 10, unit: 'MINUTES')
   }
   stages {
-    // stage('prepare') {steps {echo 'prepare'}}
+    stage('Build Container') {
+      environment {
+        QUAY_CREDS=credentials('candlepin-quay-bot')
+      }
+      steps {
+        sh './containers/build_and_push.sh'
+      }
+    }
     stage('Test') {
       parallel {
-        stage('Python stylish') {
+        stage('stylish') {
           steps {
-            sh readFile(file: 'jenkins/python3-stylish-tests.sh')
+            sh (
+              script: './jenkins/toolbox-run.sh stylish jenkins/stylish.sh',
+              returnStatus: false
+            )
           }
         }
-        stage('Fedora tito') {
-          agent { label 'rpmbuild' }
-          steps { sh readFile(file: 'jenkins/tito-tests.sh') }
+        stage('tito') {
+          steps {
+            sh (
+              script: './jenkins/toolbox-run.sh tito jenkins/tito.sh',
+              returnStatus: false
+            )
+          }
         }
         // TODO: figure if this is needed and implement
```

Comment:
Is this TODO still valid?

Reply:
I think we do not need it, so I've removed it  :)